templates/repo/settings/options.tmpl

```diff
@@ -682,7 +682,7 @@
 				<div class="inline fields">
 					<label>{{.locale.Tr "repo.settings.admin_indexer_commit_sha"}}</label>
```

Comment:
Maybe we should include the label

Reply:
You mean this?
![image](https://github.com/go-gitea/gitea/assets/50507092/6c4f8d0f-21a7-4a3d-8018-5c6419d2e1e7)

Reply:
Yes

Reply:
done cfd1e432517d5b5d6d91bad5f2c5530b9b135cdd